files/build_templates/sonic_debian_extension.j2

```diff
@@ -610,6 +610,8 @@ sudo cp $files_path/$ISSU_VERSION_FILE $FILESYSTEM_ROOT/etc/mlnx/issu-version
 sudo cp $files_path/$MLNX_FFB_SCRIPT $FILESYSTEM_ROOT/usr/bin/mlnx-ffb.sh
 sudo cp $files_path/$MLNX_ONIE_FW_UPDATE $FILESYSTEM_ROOT/usr/bin/$MLNX_ONIE_FW_UPDATE
 sudo cp $files_path/$MLNX_SSD_FW_UPDATE $FILESYSTEM_ROOT/usr/bin/$MLNX_SSD_FW_UPDATE
+sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/$MLNX_PHY_DEBUG_TOOLS || \
```

Comment:
@noaOrMlnx can we avoid this? Looks like this should be enough:
`$(SONIC_ONE_IMAGE)_INSTALLS += ... $(MLNX_PHY_DEBUG_TOOLS)`

Reply:
@nazariig 
removed